Answers.md

```diff
@@ -1 +1,20 @@
+Describe the biggest difference between .forEach & .map.
 
+forEach acts like a for loop and allows you to interate through each item in a array. Map just orders items into an app.
```

Comment:
I think you're on the right track. `forEach` always you to iterate through an array and execute a function or each element if you choose to do so. You also do not mutate the array at all. 

`map` also allows you to iterate through the array changing the values in some way. `map` also does not mutate the original array, but instead creates a new one.